plugin.xml

```diff
@@ -9,7 +9,7 @@
     <clobbers target="cordova.plugins.OSFirebasePerformance"/>
   </js-module>
 
-  <dependency id="cordova-plugin-firebase-analytics" url="https://github.com/OutSystems/cordova-plugin-firebase-analytics.git#outsystems"/>
+  <dependency id="cordova-plugin-firebase-analytics" url="https://github.com/OutSystems/cordova-plugin-firebase-analytics.git#fix/remove-hook-unzipAndCopy"/>
```

Comment:
Shouldn't we point it to a specific version of the plugin?

Reply:
Same here as in the other PR: https://github.com/OutSystems/cordova-plugin-firebase-crash/pull/5

Reply:
Same answer :)